packages/enr/src/util.ts

```diff
@@ -32,5 +32,8 @@ export function bigintToBytes(n: bigint): Uint8Array {
 }
 
 export function bytesToBigint(bytes: Uint8Array): bigint {
+  if (bytes.length === 0) {
+    return 0n;
```

Comment:
Small thing, may be better to rather do BigInt(0) which has a nicer failure mode for incompatible browsers/envs

Reply:
:heavy_check_mark: